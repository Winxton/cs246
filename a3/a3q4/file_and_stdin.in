game 10 5 a1.txt stdin
	0 
	5
game 7 4 stdin b1.txt
	3
game 5 3 stdin stdin
	10
	2
game 150 150 b3.txt a1.txt
quit
