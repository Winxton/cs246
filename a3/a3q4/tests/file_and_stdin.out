Total is 10, max deduction is 5
Player A's move
Total is 7
Player B's move
Total is 6
Player A's move
Total is 3
Player B's move
Total is -2
Player B wins
Score is
A 0
B 1
Total is 7, max deduction is 4
Player B's move
Total is 6
Player A's move
Total is 3
Player B's move
Total is -1
Player B wins
Score is
A 0
B 2
Total is 5, max deduction is 3
Player A's move
Total is 2
Player B's move
Total is 0
Player B wins
Score is
A 0
B 3
Total is 150, max deduction is 150
Player B's move
Total is 147
Player A's move
Total is 7
Player B's move
Total is 4
Player A's move
Total is -146
Player A wins
Score is
A 1
B 3
